index.html

```diff
@@ -70,5 +70,314 @@ <h1 class="headline_title">
+                     </li>
+                     <li class="irem">
+                         <button class="btn1">
+                             JavScript
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn1">
+                             html
+                         </button>
+                     </li>
+                 </ul>
+                 <button class="btngreen">
+                     See Project
+                 </button>
+             </div>
+         </div>
+
+         <div class="imgplacesholder"></div>
+             <div>
+                 <h2 class="engin">
+                     Multi-Post Stories <br> Gain+Glory
+                 </h2>
+                 <ul class="eylem">
+                     <li class="irem">
+                         <button class="btn1">
+                             Ruby on rails
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn">
+                             css
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn1">
+                             JavScript
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn1">
+                             html
+                         </button>
+                     </li>
+                 </ul>
+                 <button class="btngreen">
+                     See Project
+                 </button>
+             </div>
+         </div>
+
+         <div class="imgplacesholder"></div>
+             <div>
+                 <h2 class="engin">
+                     Multi-Post Stories <br> Gain+Glory
+                 </h2>
+                 <ul class="eylem">
+                     <li class="irem">
+                         <button class="btn1">
+                             Ruby on rails
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn">
+                             css
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn1">
+                             JavScript
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn1">
+                             html
+                         </button>
+                     </li>
+                 </ul>
+                 <button class="btngreen">
+                     See Project
+                 </button>
+             </div>
+         </div>
+        </div>
+     </section>
+
+     <section id="about-me">
+        <h1>About me</h1>
```

Comment:
For this `h1` element, kindly make the following changes:

- [ ] Convention and HTML best practices state that only one `h1` tag should be used on a webpage. You have used the h1 tag in the Headline section for the element with the text "Hey there. I'm Cansu'. Kindly change this tag to another `hx` tag between `h2-h6`. 💻 
- [ ] The second change has to do with styling, but I will add the comment here so that it can be coherent. I noticed the `background-image` for this section is not reaching the extreme end on the right side. There is a white patch to the right. Kindly reposition the background image so that it becomes identical to the Figma template you chose. 👍 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/205453249-b14b804a-f677-4275-aa7c-71756e801cfb.png)

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/205453271-a6aa660f-182d-4d87-a828-4033e0d3332e.png)